lib/ansible/modules/network/meraki/meraki_admin.py

```diff
@@ -117,6 +119,28 @@
     org_name: YourOrg
     state: absent
     email: jane@doe.com
+
+- name: Create a new administrator with full access to a tag
+  meraki_admin:
+    auth_key: abc12345
+    org_name: YourOrg
+    state: present
+    name: Jane Doe
+    orgAccess: read-only
+    email: jane@doe.com
+    tags:
+        - {"tag": "tenant", "access": "full"}
+
+- name: Create a new administrator with full access to a network
+  meraki_admin:
+    auth_key: abc12345
+    org_name: YourOrg
+    state: present
+    name: Jane Doe
+    orgAccess: read-only
+    email: jane@doe.com
+    networks:
+        - {"id": "N_12345", "access": "full"}
```

Comment:
Since both tags and networks can be a list, it would be helpful to make one of these examples show that (for example, showing two tags or two networks)

Reply:
I moved the example from having one to two items.